README.md

```diff
@@ -107,3 +107,14 @@ Many thanks to the macOS community and special thanks to [mczachurski](https://g
 
 ## License
 [MIT](LICENSE)
+
+## Translation
+
+Equinox is translated to `English` & `French`. To translate Equinox to another language:
```

Comment:
nit: better to structure all languages as a list because it would easier to add new languages.
```
Equinox is translated to:
- English
- French
```

Reply:
Good idea, can you try and commit this suggestion :
```suggestion
Equinox is translated to:
- English
- French, by @W1W1-M

To translate Equinox to another language:
```

Reply:
Committed myself !